lambdas/webhook.py

```diff
@@ -15,4 +16,34 @@ def lambda_handler(event: dict, context: object) -> str:
             "Sentry DSN found, exceptions will be sent to Sentry with env=%s", env
         )
 
-    return "Hello lambda!"
+    logger.debug(json.dumps(event))
+
+    base_response = {
+        "headers": {"Content-Type": "text/plain"},
+        "isBase64Encoded": False,
+    }
+
+    request_type = event["requestContext"]["http"]["method"]
+    if request_type == "GET":
+        result = handle_get_request(event)
+    else:
+        result = {
+            "statusCode": 405,
+            "body": "HTTP method not allowed. Only GET requests are supported.",
+        }
+
+    return base_response | result
+
+
+def handle_get_request(event: dict) -> dict[str, object]:
+    try:
+        challenge_secret = event["queryStringParameters"]["challenge"]
```

Comment:
Just confirming, there's no validation that happens with the `challenge` parameter? Any value works?

Reply:
Correct, that gets configured (by us) in Alma, and all our webhook listener needs to do is send back exactly what Alma sent us.